modules/accessing-an-example-cluster-node-tuning-operator-specification.adoc

```diff
@@ -22,5 +22,5 @@ The default CR is meant for delivering standard node-level tuning for the {produ
 
```

Comment:
I know it's beyond the scope of this PR, but line 14 could use a revision. First, it uses a numbered step (because it's a 1-step procedure, it should be a bullet point); second, it should be "Run the following command:" So it should be written as: 

* Enter the following command: 

or 

* Run the following command: 

You could even add "to access an example Node Tuning Operator specification" to the commands I've written above if you wanted.